src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,50 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int ARRAY_SIZE = 10;
+
+    private final K[] keyArr = (K[])new Object[ARRAY_SIZE];
+    private final V[] valArr = (V[])new Object[ARRAY_SIZE];
+    private int pos = 0;
```

Comment:
```suggestion
    private int size;
```

0 if default value for int